src/components/CardTable/CardTable.jsx

```diff
@@ -172,6 +178,9 @@ export default class CardTable extends Component {
     })
 
   handleWindowResize = () => new Promise((resolve) => {
+    if (!this.mounted) {
+      throw new Error('Not mounted')
+    }
```

Comment:
This one isn't 100% necessary but given the setState is in a promise i think we should have some checking. This shouldn't ever really be called in an unmounted component but in some edge cases (when the application quickly added and removed the table) it would occasionally be called with null. It doesn't occur in normal usage anymore but I can see it coming back.